lib/Net/SAML2/IdP.pm

```diff
@@ -225,14 +228,18 @@ around BUILDARGS => sub {
         for my $use (keys %{$params{certs}}) {
             my $certs = $params{certs}{$use};
             for my $pem (@{$certs}) {
+                my @errors;
```

Comment:
I think you want this outside of the loop? If not, it doesn't need to be an array. It doesnt need exist at all. See my point further down.